librocksdb_sys/src/lib.rs

```diff
@@ -273,6 +273,10 @@ extern "C" {
                                        fairness: i32)
                                        -> *mut DBRateLimiter;
     pub fn crocksdb_ratelimiter_destroy(limiter: *mut DBRateLimiter);
+    pub fn crocksdb_options_set_soft_pending_compaction_bytes_limit(options: *mut DBOptions,
```

Comment:
Both declarations should be moved up to get close to those `crocksdb_options_xxx` declarations.